projects/detr3d/configs/detr3d_res101_gridmask_cbgs.py

```diff
@@ -0,0 +1,80 @@
+_base_ = ['./detr3d_res101_gridmask.py']
```

Comment:
for convention, we usually use r101 to represent resnet-101, so we can change res101 to r101